<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Telik v1.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1041</width>
      <height>651</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tab">
        <property name="accessibleName">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <attribute name="title">
         <string>Параметры</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>641</width>
           <height>474</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>110</height>
             </size>
            </property>
            <property name="title">
             <string>Параметры эксперимента</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <widget class="QWidget" name="formLayoutWidget_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>541</width>
               <height>58</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Файл конфигурации</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QToolButton" name="toolButton">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_4">
                <property name="placeholderText">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Название эксперимента</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" colspan="2">
               <widget class="QLineEdit" name="lineEdit_2">
                <property name="toolTip">
                 <string>Введите название эксперимента (используйте только буквы и цифры)</string>
                </property>
                <property name="text">
                 <string>Эксперимент_</string>
                </property>
                <property name="placeholderText">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="title">
             <string>Управление осями</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <widget class="QSpinBox" name="spinBox">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>60</y>
               <width>111</width>
               <height>51</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="minimum">
              <number>-999999999</number>
             </property>
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
            <widget class="QSpinBox" name="spinBox_2">
             <property name="geometry">
              <rect>
               <x>250</x>
               <y>60</y>
               <width>101</width>
               <height>51</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="minimum">
              <number>-999999999</number>
             </property>
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
            <widget class="QSpinBox" name="spinBox_3">
             <property name="geometry">
              <rect>
               <x>360</x>
               <y>60</y>
               <width>111</width>
               <height>51</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="minimum">
              <number>-999999999</number>
             </property>
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>60</y>
               <width>101</width>
               <height>51</height>
              </rect>
             </property>
             <property name="text">
              <string>Переместить
все оси</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_3">
             <property name="geometry">
              <rect>
               <x>150</x>
               <y>120</y>
               <width>75</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>Текущее</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_4">
             <property name="geometry">
              <rect>
               <x>260</x>
               <y>120</y>
               <width>75</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>Текущее</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_5">
             <property name="geometry">
              <rect>
               <x>380</x>
               <y>120</y>
               <width>75</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>Текущее</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_7">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>40</y>
               <width>47</width>
               <height>13</height>
              </rect>
             </property>
             <property name="text">
              <string>X</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_8">
             <property name="geometry">
              <rect>
               <x>280</x>
               <y>40</y>
               <width>47</width>
               <height>13</height>
              </rect>
             </property>
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_9">
             <property name="geometry">
              <rect>
               <x>390</x>
               <y>40</y>
               <width>47</width>
               <height>13</height>
              </rect>
             </property>
             <property name="text">
              <string>Z</string>
             </property>
            </widget>
            <widget class="QSlider" name="verticalSlider_2">
             <property name="geometry">
              <rect>
               <x>590</x>
               <y>40</y>
               <width>20</width>
               <height>121</height>
              </rect>
             </property>
             <property name="maximum">
              <number>5000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="invertedAppearance">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QLabel" name="label_16">
             <property name="geometry">
              <rect>
               <x>530</x>
               <y>50</y>
               <width>47</width>
               <height>13</height>
              </rect>
             </property>
             <property name="text">
              <string>Speed</string>
             </property>
            </widget>
            <widget class="QSpinBox" name="spinBox_4">
             <property name="geometry">
              <rect>
               <x>510</x>
               <y>70</y>
               <width>71</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="minimum">
              <number>-999999999</number>
             </property>
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
            <widget class="QLabel" name="label_17">
             <property name="geometry">
              <rect>
               <x>590</x>
               <y>160</y>
               <width>21</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="label_18">
             <property name="geometry">
              <rect>
               <x>580</x>
               <y>20</y>
               <width>41</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>5000</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>150</height>
             </size>
            </property>
            <property name="title">
             <string>Другие параметры</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <widget class="QPushButton" name="pushButton_6">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>90</y>
               <width>211</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>Переместить все оси в &quot;0&quot;</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_7">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>50</y>
               <width>151</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Переместить ось X в &quot;0&quot;</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_8">
             <property name="geometry">
              <rect>
               <x>200</x>
               <y>50</y>
               <width>151</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Переместить ось Y в &quot;0&quot;</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_9">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>360</x>
               <y>50</y>
               <width>151</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>Переместить ось Z в &quot;0&quot;</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="pushButton_12">
         <property name="geometry">
          <rect>
           <x>700</x>
           <y>20</y>
           <width>101</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Старт</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_13">
         <property name="geometry">
          <rect>
           <x>700</x>
           <y>80</y>
           <width>101</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>Стоп</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Эксперимент</string>
        </attribute>
        <widget class="QWidget" name="formLayoutWidget">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>591</width>
           <height>381</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout"/>
        </widget>
        <widget class="QPushButton" name="pushButton_10">
         <property name="geometry">
          <rect>
           <x>650</x>
           <y>100</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>append</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="geometry">
          <rect>
           <x>650</x>
           <y>30</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="geometry">
          <rect>
           <x>650</x>
           <y>60</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_11">
         <property name="geometry">
          <rect>
           <x>650</x>
           <y>210</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Scroll</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_6">
         <property name="geometry">
          <rect>
           <x>640</x>
           <y>170</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="title">
          <string>Положение оси X, имп:</string>
         </property>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>161</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1857</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>21</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>T2</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>21</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>T1</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_18">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>110</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>PWR</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="title">
          <string>Положение оси Y, имп:</string>
         </property>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>161</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="text">
           <string>546</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buddy">
           <cstring>spinBox</cstring>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>21</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>T1</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>21</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>T2</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_19">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>110</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>PWR</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="title">
          <string>Положение оси Z, имп:</string>
         </property>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>161</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="text">
           <string>12932</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>21</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>T1</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>21</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>T2</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_20">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>110</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>PWR</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="title">
          <string>Показания АЦП, мВ:</string>
         </property>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>181</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="text">
           <string>452,54</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
